src/npcmove.cpp

```diff
@@ -1316,7 +1316,7 @@ npc_action npc::method_of_attack()
             return m.melee() || m.flags.count( "NPC_AVOID" ) ||
                    !m->ammo_sufficient( m.qty ) || !can_use( *m.target ) ||
                    m->get_gun_ups_drain() > ups_charges ||
-                   ( ( danger <= ( m.qty == 1 ? 3.0 : 15.0 ) ) && !emergency() ) ||
+                   ( ( danger <= ( m.qty == 1 ? 0.0 : 0.5 ) ) && !emergency() ) ||
```

Comment:
Now NPCs will use automatic weapons fire against any enemy, which a lot of people don't want.

Even if something like this change were a good idea, it should be:
```suggestion
                   ( ( danger <= ( m.qty == 1 ? 0.0 : 15 ) ) && !emergency() ) ||
```
so they will use ranged weapons against all enemies, and automatic fire against dangerous enemies.

Reply:
I am all for that. I did not understand enough to suggest that solution, thank you.